nixos/tests/gnome-photos.nix

```diff
@@ -0,0 +1,24 @@
+# run installed tests
+import ./make-test.nix ({ pkgs, lib, ... }:
+
+{
+  name = "gnome-photos";
+  meta = {
+    maintainers = pkgs.gnome-photos.meta.maintainers;
+  };
+
+  machine = { pkgs, ... }: {
+    imports = [ ./common/x11.nix ];
+    programs.dconf.enable = true;
+    services.gnome3.at-spi2-core.enable = true; # needed for dogtail
+    environment.systemPackages = with pkgs; [ gnome-desktop-testing ];
+    services.dbus.packages = with pkgs; [ gnome-photos ];
+  };
+
+  testScript = ''
+    $machine->waitForX;
+    # dogtail needs accessibility enabled
+    $machine->succeed("env GIO_EXTRA_MODULES='${lib.getLib pkgs.gnome3.dconf}/lib/gio/modules' XDG_DATA_DIRS='${pkgs.gsettings-desktop-schemas}/share/gsettings-schemas/${pkgs.gsettings-desktop-schemas.name}' ${pkgs.glib.bin}/bin/gsettings set org.gnome.desktop.interface toolkit-accessibility true 2>&1");
```

Comment:
Can you make `let` the environment variables string into the `testScript` so it's clearer to read?